compiler/CompilerConfig.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
You have both CompilerConfig.yaml and compiler/CompilerConfig.yaml.
So which one is actually used?
It may lead to some confusions.

Reply:
Fix this issue - only one of them left (in compiler directory)